cogs/annual_handover_and_reset.py

```diff
@@ -88,7 +87,7 @@ async def committee_handover(self, ctx: TeXBotApplicationContext) -> None:
         await initial_response.edit(
             content=(
                 ":hourglass: Giving committee users access to the #handover channel and "
-                "removing the \"Committee\" role... :hourglass:"
+                'removing the "Committee" role... :hourglass:'
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes